main.tf

```diff
@@ -2,71 +2,82 @@ provider "metal" {
   auth_token = var.auth_token
 }
 
+locals {
+  ssh_user               = "root"
+  timestamp_sanitized    = replace(timestamp(), "/[-| |T|Z|:]/", "")
+  project_name_sanitized = replace(var.project_name, "/[ ]/", "_")
+  ssh_key_name           = format("%s-%s-key", local.project_name_sanitized, local.timestamp_sanitized)
```

Comment:
A filename based on the time will jitter on `plan`/`apply`. 

We can use the pattern from the Anthos on Baremetal project: https://github.com/equinix/terraform-metal-anthos-on-baremetal/blob/master/main.tf#L30

Reply:
Yeah. I agree. I'll have to create a random string for this as we don't have a user input that is randomized like the baremetal version.

Reply:
Done.